index/scorch/segment_plugin.go

```diff
@@ -23,13 +23,15 @@ import (
 	zapv12 "github.com/blevesearch/zap/v12"
 	zapv13 "github.com/blevesearch/zap/v13"
 	zapv14 "github.com/blevesearch/zap/v14"
+	zapv15 "github.com/blevesearch/zap/v15"
 )
 
 var supportedSegmentPlugins map[string]map[uint32]segment.Plugin
 var defaultSegmentPlugin segment.Plugin
 
 func init() {
 	ResetPlugins()
+	RegisterPlugin(zapv15.Plugin(), false)
```

Comment:
Am I reading this correctly that zap v11 is still used by default? How does one tell bleve to use v15?

Reply:
Yes, we made a decision earlier that the defaults are essentially part of the API, so changing the file format would be equivalent to a breaking change.

We are [planning bleve 2.x now](https://github.com/blevesearch/bleve/issues/1495), and scorch will become the default index type, and the latest zap will also become the default at that time.

One can force the segment type and version used with the `forceSegmentType` and `forceSegmentVersion` config settings.  They are described here: https://github.com/blevesearch/bleve/issues/1350